<!DOCTYPE html>
<html lang="en-US">

    <head>

        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <meta http-equiv="content-type" content="text/html; charset=utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">

        
        <meta name="description" content="Simple minimalist theme">
        
        <meta name="keywords" content="minimalist,blog,goa,hugo,developer">

        <title>
              Alex Camp Reis - Regulating many genes with CRISPR and ELSAs 
        </title>

        <meta name="generator" content="Hugo 0.68.3" />

        
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/styles/default.min.css">
        

        <link rel="stylesheet"
            href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400&family=Roboto+Slab:wght@400;700&family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700" />
        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
            integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
            crossorigin="anonymous" />
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css">
        <link rel="stylesheet" href="http://alexcampreis.com/css/main.css">
        <link rel="stylesheet" href="http://alexcampreis.com/css/custom.css">

        <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
        <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
        <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
        <link rel="manifest" href="/site.webmanifest">
        <meta name="theme-color" content="#ffffff"></head>

    <body lang="en-US">
        <div class="container my-auto">

<header class="text-start title">
  <h1 class="title">Regulating many genes with CRISPR and ELSAs</h1>
</header>

<section id="category-pane" class="meta">
  
  <div class="d-flex flex-column flex-md-row justify-content-between">
    <h6 class="text-start meta">
       PUBLISHED ON AUG 1, 2020
      
      / 5 MIN READ
      
    </h6>
  </div>
  
</section>

<section id="content-pane">
  <div class="text-justify content">
    
    <nav id="TableOfContents">
  <ul>
    <li><a href="#references">References</a></li>
  </ul>
</nav>
    
    <p>Synthetic biologists, our group included, have been drawn by the power of CRISPR technologies<sup>1</sup> to completely revolutionize how we program cell behavior, for example, by designing programmable CRISPR-based genetic circuits. Genetic circuits, like digital ones, take input signals from the surrounding environment (e.g. relevant biologics, chemicals, temperature, UV), and through a series of logic gates, perform computation or “signal processing”, with useful output that can be read out or passed to biological actuators within a programmed cell. To date, most genetic circuits have been built using transcription factors (TFs), which have inherent limitations: (1) only a handful of orthogonal TFs have been characterized<sup>2</sup>, (2) proteins require a lot of encoding space (i.e. more DNA), and (3) TFs are toxic when expressed at too high of a protein level. These issues have generally limited TF-based genetic circuits to about 10 regulators, where each regulator emulates a logic function (e.g. a NOT-gate)</p>
<p>In our first year of graduate school, during our synthetic biology graduate course, we were brainstorming novel types of genetic circuits to work on for a collaborative side-project. We came up with an idea for a large-scale analog genetic circuit, with potentially thousands of regulators, to be implemented in Escherichia coli. Even to this day, this scale and complexity has not been realized in an engineered biological program, but we realized CRISPR could help get us there. A CRISPR-based circuit would be relatively small in construct size, as only one protein “master” regulator (dCas9) is required to be co-expressed with each sgRNA regulator. We also noted that if the guides were designed to be orthogonal to the genome, off-target binding could be largely mitigated, minimizing cell toxicity. There seemed to be an issue however – most CRISPR based systems at the time had only 2-3 co-expressed sgRNAs, with few examples expressing up to 12 sgRNAs. The question was: What limits the scalability of CRISPR systems?</p>
<p>To address this, we conducted a “red teaming” exercise<sup>3</sup> which involved simulating the design, synthesis, assembly, integration and expression of the proposed system to identify project failure points. One core design aspect stood out as the primary issue: Repetitive DNA causes problems.</p>
<p>In order to build our many-regulator CRISPR circuit, we wanted to order large arrays of sgRNAs on gBlock Gene Fragments from Integrated DNA Technologies, Inc. (IDT). Hierarchical assembly of individual sgRNAs using cloning methods such as Golden Gate or Gibson assembly was going to require too much cloning time and would generally limit the scale of the sgRNA array that we could build. Instead, we recognized that we could fit 13 sgRNAs on a 3 kb gBlock (the maximum size offered by IDT), and if we could order a series of gBlocks to assemble together, we could quickly be building systems with many co-expressed sgRNAs. One issue here though – gene synthesis companies couldn’t synthesize this design because of the repeated genetic elements (promoters, sgRNA handles, terminators). Existing gene fragment synthesis technology generally relies on assembly via hybridization between many small oligonucleotides, and if these oligos are repetitive, they mis-hybridize, and synthesis fails.</p>
<p>Intriguingly, during the red teaming exercise, we recognized another related failure point: repetitive DNA is susceptible to homologous recombination in vivo. In E. coli, the minimal length required for recombination to efficiently occur is approximately 20 base pairs<sup>4</sup>. We acknowledged that our engineered CRISPR systems could be a resource burden or interfere with native host processes, potentially introducing a selective phenotype, which then over time would most likely recombine to delete portions of our arrays. Repetitive DNA in designed genetic systems has been the cause of failure in a few reported cases, in other cases reported to our group but never mentioned during publication, and in potentially many more undiagnosed cases.</p>
<p>With these challenges in mind, we compiled and characterized a non-repetitive parts library including all of the regulatory elements needed for sgRNA array design: constitutive promoters, sgRNA handles, transcriptional terminators, and biologically neutral spacers (Fig. 1a). The most significant challenge here was to design unique sgRNA handles that performed equally as well as the wild-type handle. Work had been previously done to identify some of the sequence-structure features required for Cas9-handle recognition<sup>5,6</sup>, but no comprehensive mutagenesis study had been conducted. We tested non-repetitive sgRNA handles across 3 design-build-test-learn cycles using an in vivo mRFP1 reporter knockdown assay and machine learning to iteratively improve the design constraint, resulting in 28 highly functional non-repetitive handles for CRISPR-Cas9 applications.</p>
<p>We then designed and thoroughly characterized 3 “extra-long sgRNA arrays” (ELSAs), with up-to 22 coexpressed sgRNAs in E. coli, for various demonstrative applications including: a succinate producing strain via metabolic rewiring, a biocontainment strain with inducible multi-auxotrophy, and an antibiotic susceptible strain with reduced persister cell survival (Fig. 1b-c). Notably, to achieve synthesis success and genetic stability of these ELSAs, we developed and used an automated design algorithm, referred to as ELSA Calculator that combines the non-repetitive parts to have a maximum DNA repeat length (e.g. L=12 bp) and to satisfy 23 design rules pertaining to synthesis and stability (link below).</p>
<p>This work is the first of many steps towards realizing large scale CRISPR systems. We hope to soon see even larger toolboxes of non-repetitive sgRNA handles for both S. pyogenes Cas9, as well as other Cas proteins and CRISPR systems. Of note, the final design constraint for the sgRNA handle encompasses a design space including an estimated hundreds of thousands of non-repetitive handles that could be co-expressed! We’re excited to see how other researchers across the life sciences make use of ELSAs and we’re excited to see the future of non-repetitive DNA design in synthetic biology.</p>
<h2 id="references">References</h2>
<ol>
<li>
<p>Doudna, J.A. &amp; Charpentier, E. The new frontier of genome engineering with CRISPR-Cas9. Science 346, 1258096 (2014).</p>
</li>
<li>
<p>Stanton, B.C. et al. Genomic mining of prokaryotic repressors for orthogonal logic gates. Nature chemical biology 10, 99 (2014).</p>
</li>
<li>
<p>Hoffman, B.G. Red Teaming: How Your Business Can Conquer the Competition by Challenging Everything. (Crown business, 2017).</p>
</li>
<li>
<p>Shen, P. &amp; Huang, H.V. Homologous recombination in Escherichia coli: dependence on substrate length and homology. Genetics 112, 441-457 (1986).</p>
</li>
<li>
<p>Briner, A.E. et al. Guide RNA functional modules direct Cas9 activity and orthogonality. Molecular cell 56, 333-339 (2014).</p>
</li>
<li>
<p>Nishimasu, H. et al. Crystal structure of Cas9 in complex with guide RNA and target DNA. Cell 156, 935-949 (2014).</p>
</li>
</ol>

  </div>
</section>

<section id="tag-pane" class="meta">
  
  <div class="d-flex flex-column flex-md-row justify-content-between">
    <h6 class="text-end meta">
      
    </h6>
  </div>
  
</section>







<section id="menu-pane" class="menu text-center">
  
  <nav aria-label="Page navigation">
    <ul class="pagination justify-content-center">
      
      
      <li class="menu-item">
        <a class="menu-item" href="http://alexcampreis.com/posts/post1/">&lt; prev</a>
      </li>
      

      <li class="menu-item">
        
        <span class="menu-item">&nbsp;|&nbsp;</span>
        
        <a class="menu-item" href="/posts">posts</a>
        
        <span class="menu-item">&nbsp;|&nbsp;</span>
        
      </li>

      
      <li class="menu-item">
        <a class="menu-item" href="http://alexcampreis.com/posts/post3/">next &gt;</a>
      </li>
      
      
    </ul>
  </nav>
  
  <h4 class="text-center mt-3"><a class="menu-item" href="http://alexcampreis.com/">home</a></h4>
</section>

<footer class="text-center footer">
  <hr />
  
  <h6 class="text-center copyright">© 2024. Alex Reis. <a href="http://creativecommons.org/licenses/by/3.0/">Some Rights Reserved</a>.</h6>
  
  
  

</footer>
</div>



<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/highlight.min.js"></script>


<script>hljs.highlightAll();</script>




<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
  integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
<script src="/js/custom.js"></script>
</body>

</html>

